docs/docs/reference/singleton-types.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc-page
+title: "Literal Singleton Types"
+---
+
+Singleton types is one of the most popular features in Scala. Literal Singleton types takes it a step further.
+Right now, a popular way to replicate literal singleton types is to create macros and/or use libraries such as
```

Comment:
> Right now

Doc should be timeless when possible.

If you want to cite shapeless here please do it more precisely by pointing to the corresponding documentation.